### ISSUE TITLE
"Error updating the share" fails when trying to set "Allow editing"

### ISSUE THREAD

**Issue by Wikinaut** [ONLYOFFICE/CommunityServer#191]
crossposted here: https://github.com/ONLYOFFICE/onlyoffice-nextcloud/issues/223

**Comment by Wikinaut** [comment]
https://github.com/ONLYOFFICE/onlyoffice-nextcloud/issues/223

**Comment by Wikinaut** [comment]
![grafik](https://user-images.githubusercontent.com/1151915/72962841-c8797780-3db5-11ea-86dc-e9def0bc9f7c.png)

**Comment by ShockwaveNN** [comment]
This issue is not related to CommunityServer

**Comment by Wikinaut** [comment]
Solution:

It appears, that this setting (1) **must** be active:
![grafik](https://user-images.githubusercontent.com/1151915/72977063-fb326880-3dd3-11ea-9eb6-fd439e7b330e.png)

**Comment by ShockwaveNN** [comment]
@Wikinaut This issue tracker for completly different product - `ONLYOFFICE Community Server`, it's not related to `Nextcloud Community Document Server` in any way

**Comment by Wikinaut** [comment]
uh, was not clear before. Now I know. Where is this located?

**Comment by ShockwaveNN** [comment]
> Where is this located?

What's `this`?

If you mean there is `Nextcloud Community Document Server` support - you got comment here: https://github.com/ONLYOFFICE/onlyoffice-nextcloud/issues/223#issuecomment-577612951

**Comment by Wikinaut** [comment]
https://github.com/ONLYOFFICE/onlyoffice-nextcloud/issues/223#issuecomment-577612951 is something **different**. I mean this https://apps.nextcloud.com/apps/documentserver_community

**Comment by Wikinaut** [comment]
tl;dr

The correct place to file OMLYOFFICE/Document Server issues for Nextcloud 18 users is:

https://github.com/nextcloud/documentserver_community/issues

**Comment by ShockwaveNN** [comment]
> The correct place to file OMLYOFFICE/Document Server issues for Nextcloud 18 users is:

No, this is not right

`Community Document Server` and `ONLYOFFICE integration app` is different product

`Community Document Server`  is product of Nextcloud which use ONLYOFFICE DocumentServer as base app
`ONLYOFFICE integration app` is product of ONLYOFFICE which use ONLYOFFICE DocumentServer as base app

**Comment by Wikinaut** [comment]
I have no idea what the difference is. I use
* Nextcloud 18 with
* https://apps.nextcloud.com/apps/onlyoffice
* https://apps.nextcloud.com/apps/documentserver_community

**Comment by ShockwaveNN** [comment]
tl, dr:

If you use https://apps.nextcloud.com/apps/documentserver_community you should direct all question to Nextcloud support (https://github.com/nextcloud/documentserver_community/issues)
